common/lib/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer.rb

```diff
@@ -45,6 +45,8 @@ def sanitize_links_and_mentions(text:, unsafe: false)
           sanitize_nwo_text(doc)
 
           mode = unsafe ? :UNSAFE : :DEFAULT
+          return doc.to_commonmark([mode] + COMMONMARKER_OPTIONS) unless format_html
```

Comment:
Actually, if we do this, it'll affect all providers that don't support html:
https://github.com/dependabot/dependabot-core/blob/9df8aec3878641a139b9b63d94e43dbbb125c715/common/lib/dependabot/pull_request_creator/message_builder/metadata_presenter.rb#L244

That doesn't mean this is a bad way to go, but let me double-check if that list is still accurate... for example I already found #6453

Reply:
Let's go ahead and merge this, and then I'll open a follow-up PR to address #6453 and get input from Microsoft folks... I have a feeling that it now supports HTML, which would make our lives easy here.